test/index.js

```diff
@@ -764,6 +771,7 @@ test('traverse', function() {
   eq(Z.traverse(Identity, identInc, [1, 2, 3]), Identity([2, 3, 4]));
   eq(Z.traverse(Identity, identInc, Nil), Identity(Nil));
   eq(Z.traverse(Identity, identInc, Cons(1, Cons(2, Cons(3, Nil)))), Identity(Cons(2, Cons(3, Cons(4, Nil)))));
+  eq(Z.traverse(Lazy$of, Lazy$of, range(70000)).run(), range(70000));
```

Comment:
```js
function computeCallStack() {
  try {
    return 1 + computeCallStack()
  } catch (_) {
    return 1
  }
}

const MAX_STACK = computeCallStack()
```
stack sizes [Node: 11034, Firefox: 50994, Chrome: 10402](http://www.2ality.com/2014/04/call-stack-size.html)

In node it's size is around 10k so if we use `MAX_STACK` instead of 70k, timeout issue might go away